meetings/2021-08-18.md

```diff
@@ -0,0 +1,177 @@
+I need to create these issues to be able to link to them in order [PR 244: Effective ACR discovery](https://github.com/solid/authorization-panel/pull/244), and so for it to be merged.
+
+Matthieu: Given that we have 4 potential solutions to minimising the number of requests involved in discovering the effective ACR, is everyone ok to merge #244?
+
+Henry: Yes.
+
+Elf: +1
+
+Justin: +1
+
+Matthieu: Merged.
+
+Henry: The last solution might be useful for ACP too, if you want to weaken the requirement that every resource must have one ACR.
+
+Matthieu: I'm not sure that falls into the ACP model, but let's review once the draft ACP PR is ready for review (that will be done today).
+
+
+### Delegation use case
+
+Henry: Could you give us a review of that use case Elf?
+
+Elf: How do you feel about using the [GNAP terminology](https://www.ietf.org/archive/id/draft-ietf-gnap-core-protocol-00.html#name-roles) of Resource owner, server...?
+
+Justin: I like using that terminology because I think it's well thought out and aligns with where we're going in terms of different actors and roles and responsibilities.
+
+Henry: Btw, a bit out of context, but how long is the GNAP spec since we mentioned it (I think it was 80 pages last time I looked)?
+
+Justin: Maybe Pavlik you could extract the relevant terminology.
+
+Henry: It's fine, I need to study that more thoroughly. Justin, you maintain this document (authorization UCs) so you might know what is relevant best.
+
+Justin: The AC UCs probably don't have a huge delta, maybe Pavlik you could do a PR and see how it aligns. Maybe spend 15mn to do some side by side. My experience with GNAP is that it is going to provide more specificity about the actors.
+
+Elf: I also added a part about delegation vs. impersonation. Bob would be able to access, but from the Authorization Server's perspective, it would be Alice accessing (impersonation) with delegation, ACME systems would know that Bob is accessing. Let's take a week to process that.
+
+
+### Fixes and add story for privacy
+
+* [PR 241: Fixes and add story for privacy](https://github.com/solid/authorization-panel/pull/241)
+
+Elf: I think the musings about clients and so on distract from the main goal of focusing on inheritance. It would be good to have end to end but maybe in a primer or something like that.
+
+Henry: I'm happy to start moving stuff to another place at some point in the future. But, at present, we have only three UC evaluations. It is a lot of work to do each, and so I'd rather have one that has a large coverage to test the concepts, rather than start off doing another one.
+
+These evaluations are, in my view, meant to show how the whole system is working.
+Unless you have a client and a server working together, we don't have a protocol. We need a system where the client can discover what the control rules are.
+
+Elf: That is UC3, would you add those concerns to every use case?
```

Comment:
It's OK for some Requirement(s) to be required by multiple Use Cases.  Typically, there will be some Requirements that are required by two or more Use Cases, and some that are required by only one Use Case.  Use Cases should be realistically simple (or complicated), and not leave out important elements of process just to limit the number of Requirements they have.  It's OK for one Use Case to have several Requirements.